docs/kyma/08-07-create-volume-snapshots-providers.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Create on-demand volume snapshots for cloud providers
+type: Tutorials
+---
+
+These tutorials show how to create on-demand volume snapshots for cloud providers. Before you proceed with the tutorial, read the general instructions on [creating volume snapshots](#tutorials-create-volume-snapshots).
+
+## Create a volume snapshot for AKS
```

Comment:
Have you considered hiding AKS and GKE under tabs? 🤔

Reply:
I was thinking about it, but the there is also Gardener 🤔 but let me try this :)